src/data/markdown/translated-guides/en/02 Using k6/07 Modules.md

```diff
@@ -77,12 +77,17 @@ These libraries can either be downloaded and included with the test project or l
 
 The steps of this tutorial are as follows:
 
-- [Setting up the bundler](#setting-up-the-bundler)
-  - [Installing packages](#installing-packages)
-  - [Configuring Webpack](#configuring-webpack)
-  - [Adding a bundle command](#adding-a-bundle-command)
-- [Running the bundling](#running-the-bundling)
-- [Running the tests](#running-the-tests)
+- [Importing modules](#importing-modules)
+  - [Built-in modules](#built-in-modules)
+  - [Local filesystem modules](#local-filesystem-modules)
+  - [Remote HTTP(S) modules](#remote-https-modules)
+  - [The JSLib repository](#the-jslib-repository)
+- [Bundling node modules](#bundling-node-modules)
+  - [Setting up the bundler](#setting-up-the-bundler)
+  - [Running the bundling](#running-the-bundling)
+  - [Running the tests](#running-the-tests)
+- [Using local modules with Docker](#using-local-modules-with-docker)
+- [See also](#see-also)
```

Comment:
This seems wrong - this is index of the whole page but : 
1. is in the middle of the page
2. from the line just about is only about how to Bundle node modules  

so I feel like this should just be reverted. 

Was there a particular reason you decided to change this?

Reply:
I can't remember touching this 😞 I guess my mardown formatter might be the issue. Will fix it.